src/js/discharge-wizard/containers/GuidancePage.jsx

```diff
@@ -105,7 +105,7 @@ class GuidancePage extends React.Component {
           <a target="_blank" href={form.link} className="usa-button-primary va-button">Download Form {form.num}</a>
           <AlertBox
             content={<div>
-              <h4>Need Help?</h4>
+              <h4>Need help?</h4>
```

Comment:
Looping in @phiden to approve this consistency update (not sure if it applies here as well).
![localhost-3001-discharge-wizard-guidance](https://user-images.githubusercontent.com/16051172/32812698-3052bb86-c95b-11e7-9074-23b38515fde9.png)

Reply:
Thanks @cehsu -- we should be consistent across the board with things like 'Need help?' so appreciate the heads up!